python/tests/conftest.py

```diff
@@ -125,9 +129,65 @@ def before_record_cb(request: Request) -> Optional[Request]:
     return request
 
 
+def _sort_json(value: str) -> str:
+    """Parses JSON and serializes it again with a fixed key order."""
+    try:
+        json = simplejson.loads(value)
+        sorted_json = simplejson.dumps(json, sort_keys=True)
+        return sorted_json
+    except ValueError:
+        return value
+
+
+def _reformat_body(obj: dict) -> None:
+    """Sorts JSON objects embedded in HTTP payload by their keys."""
+    body = obj["body"]
+    length = None
+    if isinstance(body, str):
+        sorted_json = _sort_json(body)
+        length = len(sorted_json)
+        obj["body"] = sorted_json
+    elif isinstance(body, dict):
+        sorted_json = _sort_json(body["string"])
+        length = len(sorted_json)
+        body["string"] = sorted_json
```

Comment:
Does this need to be assigned back to `obj["body"]["string"]`?

Reply:
Yes, we're modifying the cassette's `dict` in place.